README.md

```diff
@@ -1,2 +1,24 @@
-# hw01
-Repository for homework 1
+# Biography
+
+
+![](https://media.licdn.com/mpr/mpr/shrinknp_400_400/AAEAAQAAAAAAAAqhAAAAJDNlZGE2ZDY3LTZlYTQtNGU1OS1iZTViLThiNjAzOWRlZmI4Yg.jpg)
```

Comment:
It looks like this should be an image, but the code is not correct. You need to provide a name within the [].